server/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexRequest.java

```diff
@@ -263,6 +249,21 @@ public CreateIndexRequest mapping(Map<String, ?> source) {
         return mapping(MapperService.SINGLE_MAPPING_NAME, source);
     }
 
+    /**
+     * A specialized simplified mapping source method, takes the form of simple properties definition:
+     * ("field1", "type=string,store=true").
+     */
+    public CreateIndexRequest mappingFromSimplifiedDef(Object... source) {
```

Comment:
A suggestion -- naming it `simpleMapping` could be more concise/ readable.

Reply:
++, good idea